.travis.yml

```diff
@@ -46,3 +45,31 @@ jobs:
           packages:
             - docker-ce
       script: make docker-image
+
+    - stage: Build and Deploy
+      <<: *DEFAULTS_GO
+      before_script:
+        # this stage needs to build everything including assets file and that
+        # requires running webpack, so we need nodejs here
+        - nvm install 8
+      script:
+        # compile assets via webpack and build those into bindata_assetfs.go file
+        - make bindata_assetfs.go
+        # and now compile using bakelite for all target archs
+        - go get -u github.com/terinjokes/bakelite
```

Comment:
Although I don't expect to have breaking releases, you may prefer to fetch [a release](https://github.com/terinjokes/bakelite/releases) instead.

Reply:
Last time I checked it was not possible with `go get` and github, only packages hosted on `gopkg.in` support this, is that no longer true?

Reply:
You can use dep's [`required`](https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#required). You can also fetch the tarball from GitHub, extract it, and run it from there.

Reply:
release tarball it is